proto/money/accounting/iif_export_configuration.proto

```diff
@@ -7,4 +7,6 @@ option go_package = "money/accounting";
 message IIFExportConfiguration {
 	string recurrence_descriptor = 1;
 	bool disable_asset_account_export = 2;
+	string dropbox_user_token = 3;
```

Comment:
This needs a little rework. At a minimum, you need to use the OAuth2-prescribed names for this stuff. I think your `user_token` is an access token, is it not? 

What is the `dropbox_holms_key`? Is it the client key? If so, this should be hard-coded into the codebase and not specified, as (1) client keys should be 1:1 with codebases - that's how they're intended to be used (look at own own token issuance implementation for an example of this), and (2) there's probably an accompanying consumer secret that I don't see specified here.

Can you explain what these do?